src/StoreMeta.h

```diff
@@ -30,42 +26,30 @@ typedef class StoreMeta tlv;
  * The meta data is stored using a TYPE-LENGTH-VALUE format.  That is,
  * each chunk of meta information consists of a TYPE identifier, a
  * LENGTH field, and then the VALUE (which is LENGTH octets long).
+ *
+ \par
+ * The holes in enum item values below represent deprecated/reserved IDs.
+ * \sa Store::IgnoredSwapMetaType()
  */
-enum {
-    /**
-     * Just a placeholder for the zeroth value. It is never used on disk.
-     */
-    STORE_META_VOID,
-
-    /**
-     \deprecated
-     * This represents the case when we use the URL as the cache
-     * key, as Squid-1.1 does.  Currently we don't support using
-     * a URL as a cache key, so this is not used.
-     */
-    STORE_META_KEY_URL,
-
-    /**
-     \deprecated
-     * For a brief time we considered supporting SHA (secure
-     * hash algorithm) as a cache key.  Nobody liked it, and
-     * this type is not currently used.
-     */
-    STORE_META_KEY_SHA,
+enum SwapMetaType {
+    /// Store swap metadata type with an unknown meaning.
+    /// Never used by valid stored entries.
+    STORE_META_VOID = 0,
 
```

Comment:
please leave a comment here to document that this hole is from deprecation, so it does not get trivially reused.
```suggestion

// deprecated STORE_META_KEY_URL=2
// deprecated STORE_META_KEY_SHA=3
```